### ISSUE TITLE
Minitest spec file not recognized as minitest when file contains `def`

### ISSUE THREAD

**Issue by moxley** [moxley/atom-ruby-test#57]
I have a minitest spec file that contains a method definition. `ruby-test` now thinks the file is an rspec file, and tries to run the rspec test run command.

**Comment by moxley** [comment]
```
require 'helper'

describe Object do
  it "quacks" do
    def Object.quack
      "quack"
    end
  end
end
```

**Comment by moxley** [comment]
cc @roychoo

**Comment by roychoo** [comment]
noted on this, this is an edge case, i wonder to make things simple, would it be better if we could use project configuration to specify if the project is using rspec or minitest, like how you suggested initially? instead of trying to find out what kind of test framework it is?

**Comment by moxley** [comment]
Yes, since Minitest specs and RSpec are so similar to each other, it's probably best to have a project setting to detect the framework.

**Comment by moxley** [comment]
Fixed in #67
